packages/flutter_tools/lib/src/base/file_system.dart

```diff
@@ -206,7 +208,7 @@ class LocalFileSystem extends local_fs.LocalFileSystem {
         _systemTemp?.deleteSync(recursive: true);
       }
     } on FileSystemException {
-      // ignore.
+      globals.printTrace('Failed trying to clean up temporary directory ${_systemTemp?.path}.');
```

Comment:
Logger cannot be direct injected when this class is instantiated as it is instantiated before the logger.